crates/cmd/src/main.rs

```diff
@@ -166,68 +143,80 @@ enum Subcommands {
     #[arg(short, long, value_enum)]
     guide: Guide,
     /// The certificate as a DER or PEM encoded file.
-    #[arg(short, long)]
-    path: PathBuf,
+    paths: Vec<PathBuf>,
     /// Verbose output.
-    #[arg(short, long)]
+    #[arg(short, long, conflicts_with = "quiet")]
     verbose: bool,
+    /// Quiet output: Hide all but errors/failed paths.
+    #[arg(short, long, conflicts_with = "verbose")]
+    quiet: bool,
+
+    #[arg(short, long, value_enum, default_value_t=ReportFormat::Human)]
+    /// Which format the output will be in
+    format: ReportFormat,
```

Comment:
Given that there are only two choices here and one is the default, wouldn't it be better to encode this as a `bool`? That is, just a `--json` flag given human readable output is the default.

Reply:
I thought that there may be more formats in the future, so I chose this approach to make it easier to extend (e.g. HTML-reports?).
But I don't feel strongly either way, so I can also just use a bool here, if you want.

Reply:
I am just wary of scope creep. From a maintenance point it might be easier to restrict it to those two options and any one that wants to generate additional formats can do it via wrapper scripts. Especially now that we have a structured format in JSON, tools like `jq` can help in that regard.